spiders.html

```diff
@@ -170,7 +170,7 @@
     let linkFormat = "geojson";
     function onLinkFormatChange() {
         linkFormat = document.getElementById('format-select').value
-        dataTable.draw()
+        dataTable.draw("page")
```

Comment:
What was broken and what does this do?

Reply:
When changing the link format, it resets back to page 1.

This option says to leave the ordering, search, and pagination alone. Reference https://datatables.net/reference/api/draw()

Reply:
Got it, thank you.